### PR TITLE
Fixed a static link into a href one

### PR DESCRIPTION
**Pull request by adeclosset**
Just cleaning up so belgium@gocardless.com appear on Belgium page. @samjewell (decided by my random_friend_selector) can you merge?

### CODE REVIEW

**Comment by samjewell** [comment]
Woah cool, thanks for tagging me, first time in this repo!

File: app/pages/partners/partners.fr.js [hunk 1]
```diff
@@ -76,7 +77,7 @@ export default class PartnersFr extends React.Component {
               </h2>
               <p className='u-text-s u-color-p u-margin-Vxs'>
                 Développez votre business en intégrant GoCardless. <br />
-                Pour plus d'informations, contactez-nous à <a href='mailto:france@gocardless.com'>france@gocardless.com</a>
+                Pour plus d'informations, contactez-nous à <Href to='email' className='u-link-color-p u-text-underline' />
```
Comment (samjewell) [inline app/pages/partners/partners.fr.js]: So I'm a complete noob to these pages today, and all I've done is compare with other files in the repo...
But should this also have `itemProp='email'` as an attr on the Href elem? Like this line here: https://github.com/gocardless/splash-pages/blob/dev/app/pages/faq/customers/overview/faq-customers-overview.js#L26
Either way, would you mind if I bowed out of this one, and we can get someone else to give you the thumbs up? Literally clueless!!! ;-)

Comment (samjewell) [inline app/pages/partners/partners.fr.js]: If you give yourself thumbs up then I'd offer to merge for you - but also haven't done that yet, so not sure what the process is. Is it same as before - continuously deployed from Dev to staging, and master to prod?

Comment (harrison) [inline app/pages/partners/partners.fr.js]: Thinking that it shouldn't be necessary, it's metadata for search engines. But they should be smart enough to know that `a href="mailto:xxx"` is an email so no need to tell again.. also this email isn't part of any structured contact details so not sure what google would even do with it. @Ken-Lo am I right in thinking it only makes sense in the footer where we structure all our contact deets?

Comment (Ken-Lo) [inline app/pages/partners/partners.fr.js]: Yeah, it looks like itemprop was accidentally copied & pasted from the footer - should only appear in the block in the footer.

**Comment by adeclosset** [comment]
Ok thanks all, learned something today ;) Looks all good, I'll merge!

**Comment by samjewell** [comment]
@adeclosset cheeky hint - do you want to delete the branch too?? Cheers :-)

**Comment by adeclosset** [comment]
Oops sorry didn't know it had too. Thanks for the cheeky hint ;-)

**Comment by samjewell** [comment]
@adeclosset :+1: :boom:

**Pull request merged**